icu4c/source/test/intltest/itrbnf.cpp

```diff
@@ -2818,6 +2819,15 @@ IntlTestRBNF::TestAmbiguousDelimiter() {
     }
 }
 
+void
+IntlTestRBNF::TestDividedByZero() {
+    UParseError perror;
+    UErrorCode status = U_ZERO_ERROR;
+    //RuleBasedNumberFormat rbnf(u"7060920374060940374/4:[]", Locale::getUS(), perror, status);
+    RuleBasedNumberFormat rbnf(u"7060920374060940374/4:[]", Locale::getUS(), perror, status);
```

Comment:
How does this rule result in a division by zero?

Reply:
radix=4 exponent=32 mod=0
4^32 = 2^64